### ISSUE TITLE
Failed to parse NaiveDateTime

### ISSUE THREAD

**Issue by chikko80** [poem-web/poem#436]
## Expected Behavior
Parsing of NaiveDateTime Object

## Actual Behavior
Does not parse successfully
Error message: 
```
parse request payload error: failed to parse "string(naive-date-time)": input contains invalid characters (occurred while parsing "TestStruct")
```

## Steps to Reproduce the Problem
```rust
use chrono::{NaiveDate, NaiveTime};
use poem::{listener::TcpListener, Route, Server};
use poem_openapi::{
    payload::{Json, PlainText},
    Object, OpenApi, OpenApiService,
};

struct Api;

#[derive(Object, Debug)]
#[oai(example)]
pub struct TestStruct {
    pub date: chrono::NaiveDateTime,
}

impl poem_openapi::types::Example for TestStruct {
    fn example() -> Self {
        Self {
            date: chrono::NaiveDateTime::new(
                NaiveDate::from_ymd_opt(2023, 5, 1).unwrap(),
                NaiveTime::from_hms_opt(0, 0, 0).unwrap(),
            ),
        }
    }
}

#[OpenApi]
impl Api {
    #[oai(path = "/hello", method = "post")]
    async fn index(&self, test: Json<TestStruct>) -> PlainText<String> {
        PlainText(format!("{:?}", test))
    }
}

#[tokio::main]
async fn main() -> Result<(), std::io::Error> {
    if std::env::var_os("RUST_LOG").is_none() {
        std::env::set_var("RUST_LOG", "poem=debug");
    }
    tracing_subscriber::fmt::init();

    let api_service =
        OpenApiService::new(Api, "Hello World", "1.0").server("http://localhost:3000/api");
    let ui = api_service.swagger_ui();

    Server::new(TcpListener::bind("127.0.0.1:3000"))
        .run(Route::new().nest("/api", api_service).nest("/", ui))
        .await
}
```

**Comment by sunli829** [comment]
The format must be as follows:

```rs
let dt = NaiveDate::from_ymd(2015, 9, 18).and_hms(23, 56, 4);
assert_eq!("2015-09-18T23:56:04".parse::<NaiveDateTime>(), Ok(dt));
```

**Comment by chikko80** [comment]
It's a bit confusing that the example schema is generated with "2015-09-18 23:56:04".
Is that something related to chrono or poem?
Is it possible to improve the example schema generation to generate the correct format (2015-09-18T23:56:04) for the example NaiveDateTime Object?

**Comment by sunli829** [comment]
Released in `poem-openapi@2.0.21` 🙂

**Comment by chikko80** [comment]
You are awesome!
